homework_2/cmd/main.go

```diff
@@ -4,18 +4,22 @@ import (
 	"fmt"
 
 	"code-cadets-2021/homework_2/cmd/bootstrap"
+	"code-cadets-2021/homework_2/internal/domain/services"
 	"code-cadets-2021/homework_2/internal/tasks"
 )
 
 func main() {
 	signalHandler := bootstrap.SignalHandler()
 
-	feed := bootstrap.AxilisOfferFeed()
+	feed1 := bootstrap.AxilisOfferFeed()
+	feed2 := bootstrap.AxilisSecondOfferFeed()
+	feeds := []services.Feed{feed1, feed2}
```

Comment:
Naming variables `variable1`, `variable2` is wonky and is usually a [code smell](https://en.wikipedia.org/wiki/Code_smell). It mostly points to the programmer not discovering the domain properly (e.g. if you know what information each feed provides, then you can use that to name the feeds or use any kind of distinction between the feeds to help with the naming). This is not a problem in this case, as it's just for practice and the feed data is not really something you should be concerning yourself too much with.

However, I think writing
```
feeds := []services.Feed{
    bootstrap.AxilisOfferFeed(),
    bootstrap.AxilisSecondOfferFeed(),
}
```

doesn't hide any logic of what's going on here and saves you from having to name the dummy variables.

Reply:
Just realised you need the variables in RunTasks.